CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ##### Ferenia
 
+- Added: Emmi Zone West Exit now has a Damage Boost trick to move from the center platform to the west door.
```

Comment:
With a new version out: I don't trust git that this will end up in the right section.
Can you merge main and move it manually to future 6.4.0 version?

Reply:
I believe I've merged with my main and updated the changelog.